crontab

```diff
@@ -0,0 +1,11 @@
+# Minute   Hour   DayOfMonth    Month    DayOfWeek    Command
+# (0-59)  (0-23)    (1-31)     (1-12)      (0-6)
```

Comment:
Typically used for scheduling automated builds, but can run on local MacOS as well